test/functorch/test_aotdispatch.py

```diff
@@ -4924,7 +4919,7 @@ def forward(self, arg0_1):
         self.assertExpectedInline(
             gm.true_graph_0.code.strip(),
             """\
-def forward(self, arg0_1):
+def forward(self, arg0_1, arg1_1, arg2_1):
```

Comment:
we have more args here

Reply:
sounds not an issue those are passed in the call site above but then not used in the function 
cc @bdhirsh

Reply:
hmm weirdly I tried running your PR locally and running this test, and I don't see this change. It's not immediately clear to me why the functionalization changes would affect what arguments cond() lifts during tracing

Reply:
strange?

Reply:
did build c++ again?